packages/react-core/src/components/FileUpload/examples/FileUpload.md

```diff
@@ -0,0 +1,151 @@
+---
+title: 'File upload'
+cssPrefix: 'pf-c-file-upload'
+typescript: true
+propComponents: ['FileUpload', 'FileUploadField']
+section: 'components'
+beta: true
+---
+
+import { FileUpload, Form, FormGroup, FileUploadField, Checkbox } from '@patternfly/react-core';
+
+## Examples
+
+The basic `FileUpload` component can handle simple text files via browse or drag-and-drop, loading them into memory and passing their contents as a string to an `onChange` prop.
+
+```js title=Simple-text-file isBeta
```

Comment:
I think the `isBeta` flag on the individual examples is only needed if the main component/feature is not beta. Since the entire FileUpload is currently beta, we can probably omit these in favor of the main beta flag at the top.